From 7d8886fe94726e96d2bb51466f64d70d5462c82d Mon Sep 17 00:00:00 2001
From: "Ethan D. Twardy" <ethan.twardy@gmail.com>
Date: Sun, 15 Jun 2025 12:34:46 -0500
Subject: [PATCH] [meta] libatomic-ops: Fix SRC_URI

The release is no longer available from the GitHub repository referenced
in the original SRC_URI. Update to the repository from which the fork
was created.
---
 meta/recipes-support/libatomic-ops/libatomic-ops_7.8.2.bb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta/recipes-support/libatomic-ops/libatomic-ops_7.8.2.bb b/meta/recipes-support/libatomic-ops/libatomic-ops_7.8.2.bb
index 39d761abdd..c66263b2f6 100644
--- a/meta/recipes-support/libatomic-ops/libatomic-ops_7.8.2.bb
+++ b/meta/recipes-support/libatomic-ops/libatomic-ops_7.8.2.bb
@@ -9,7 +9,7 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263 \
                     "
 
 SRC_URI = "${GITHUB_BASE_URI}/download/v${PV}/libatomic_ops-${PV}.tar.gz"
-GITHUB_BASE_URI = "https://github.com/ivmai/libatomic_ops/releases"
+GITHUB_BASE_URI = "https://github.com/bdwgc/libatomic_ops/releases"
 
 SRC_URI[sha256sum] = "d305207fe207f2b3fb5cb4c019da12b44ce3fcbc593dfd5080d867b1a2419b51"
 
-- 
2.47.2

